website/docs/job-schedulers-eks/aws-managed-airflow.md

```diff
@@ -4,9 +4,9 @@ sidebar_position: 1
 
 # Amazon Manged Workflows for Apache Airflow (MWAA)
 
```

Comment:
Let's add Introduction about MWAA with few sentences here. You can pick aws MWAA content